tools/mayan-edms/docker-compose.yml

```diff
@@ -0,0 +1,228 @@
+version: "3.9"
```

Comment:
left most of the services that came from the default. I tried removing some of them but retrieving uploaded files would fail without redis, celery or index.

Also, I've let the services be on a separate file. I think it would make the existing docker file more complex for the rest of the services if merged into only one